main/zoe/api/zoe.md

```diff
@@ -0,0 +1,77 @@
+# Zoe
+
+Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `seatAssay` and a single `escrowReceiptAssay` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `seatAssay` or `escrowReceiptAssay` and thus validate any `seat` or `escrowReceipt` they receive from someone else.
+
+### <span style="color:red">Zoe and ZoeGoverningContractFacet have some similar methods so those examples are the same. Those should be double-checked to make sure they are accurate in both locations. Methods: getSeatAssay, getEscrowReceiptAssay.</span>
+
+
+## zoe.getSeatAssay()
+- Returns: `{Assay}`
+
+Get the long-lived `seatAssay`. The mint associated with the `seatAssay` creates the ERTP payments that represent the right to claim the payouts of involvement in a contract.
+
+### <span style="color:red">Couldn't find an example in the code, this needs checking.</span>
+```js
+const someSeatAssay = zoe.getSeatAssay()
```

Comment:
line 126 of test-coveredCall.js in ERTP might be helpful here. This is meant to be about the inviteAssay:
```js
    // Bob claims all with the Zoe inviteAssay
    const inviteAssay = zoe.getInviteAssay();
    const bobExclInvitePayment = await inviteAssay.claimAll(bobInvitePayment);
```js